tex/main.tex

```diff
@@ -28,6 +28,45 @@ \section{Introduction}\label{sec:introduction}
+
+To see if one strategy is not neutral with respect to another, we want to empirically measure the fixation probability and compare to the neutral rate. To do this right we need a lot of counts, since we're estimating a binomial probability p with variance $p(1-p) / k$ and $p$ is close to $1 / N$. To get the variance small you need something like $k>1000$ observations ( we can work out the precise requirements).
+
+Note we're not estimating $r$ for each strategy (pair) since we're in a frequency dependent situation, so we need to look at the population states (1, N-1) and (N-1, 1) for every pair of strategies, i.e. we can't assume that we're in a $\rho \leftrightarrow 1-\rho$ symmetry. More precisely, $\rho_{(1, N-1)} != 1 - \rho_{(N-1, 1)}$ in general. However we can (for fun) compute $r$ from $\rho$ with Newton's method (it's not easily invertable for $N > 3$), or take a Bayesian approach on what the distribution of $\rho$ is and then compute a distribution for $r$ in the usual way.
+
+A nice addition would be, for an interesting combination of strategies, to measure the fixation value for all $(i, N-i)$ and compare to the above formula for the value of $r$ derived from the $(1, N-1)$ case. This would show how much we deviate from frequency independence.
+
+Beyond the raw data, we should try to estimate the strategies that are
+1) most resistant to invasion
+2) the best invaders
+3) "most neutral"
+
+as a function of N across the entire population of strategies. This can really open up if you want to say optimize a parameterized strategy to be most resistant to invasion (a topic of future work, perhaps) -- for example Random(p) for what p is best?
+
+The existing notebook attempts to get at 1 and 2 by looking at the distributions of fixation probabilities for each strategy -- that's what the box plots for each N try to visualize for particular N, and the "Player Rankings by Median vs. Population Size" for how the cooperative strategies become more successful as N increases. That plot is the main takeaway IMO, and reinforces the "evolution of cooperation" narrative that's so popular. We can tie back to Press and Dyson here -- yes, ZD strategies are good Head-to-Head and in small populations, but they aren't great when the population size gets bigger. How much bigger? Even at N=4 there is a dramatic decline for ZD-extort. Note that this goes against the claims of Stewart and Plotkin (they claimed that ZD strategies basically dominate the Moran process no matter how much memory you allow). This also matches our tournament results -- ZD strategies win matches but not tournaments.
```

Comment:
That's interesting. Stewart is coming to give a talk at Cardiff in December. I'll talk to him about this.